src/md/03. Test framework integration.md

```diff
@@ -1171,6 +1171,111 @@ Example of valid configuration file is below.
 </configuration>
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+ScalaTest
+--------
+
+### Installation
+* add this library to your project as dependency
+  * SBT
+   >
+   >      libraryDependencies += "com.epam.reportportal" %% "agent-scala-scalatest" % "2.6.0" % "test"
+   >
+
+  * Maven
+   >      <dependency>
+   >        <groupId>com.epam.reportportal</groupId>
+   >        <artifactId>agent-scala-scalatest_2.11</artifactId>
+   >        <version>2.6.0</version>
+   >        <scope>test</scope>
+   >      </dependency>
+
+   * Gradle
+   >
+   >      testCompile group: 'com.epam.reportportal', name: 'agent-scala-scalatest_2.11', version: '2.6.0'
+   >
+
+* add your reportportal.properties file to test/resources (See Configuration section)
+* set reporter in your build tool:
+  * _maven_ (Configuration options / Reporters): 
+  
+  http://www.scalatest.org/user_guide/using_the_scalatest_maven_plugin
+  * _sbt_
+  
+   In your build.sbt
+  ```scala
+  testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-C", "com.epam.reportportal.scalatest.RPReporter")
+  ```
+  * _gradle:_ in your test task: 
+  ```groovy
+  args = ['-C', 'com.epam.reportportal.scalatest.RPReporter']
+  ```
+* define bintray repository
+    * _sbt_
+    ```scala
+      resolvers ++= Seq(
+        "EPAM bintray" at "http://dl.bintray.com/epam/reportportal"
+      )
+    ```
+    * _maven_  
+      [Resolving Artifacts](https://bintray.com/docs/usermanual/formats/formats_mavenrepositories.html#anchorMavenResolve)  
+      URL parameter is https://dl.bintray.com/epam/reportportal/
+    * _gradle_
+    ```groovy
+    repositories {
+        jcenter()
+        mavenLocal()
+        maven { url "http://dl.bintray.com/epam/reportportal" }
+    }
+   ```
+    
+### Configuration
+
+Copy you configuration from UI of Report Portal at _User Profile_ section
+
+or
+
+In order to start using of agent, user should configure property file
+"reportportal.properties" in such format:
+
+**reportportal.properties**
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+rp.endpoint = https://rp.epam.com/
+rp.username = default
+rp.uuid = 8967de3b-fec7-47bb-9dbc-2aa4ceab8b1e
+
+rp.launch = default_TEST_EXAMPLE
+rp.project = default_project
+rp.tags = TAG1;TAG2
+
+rp.keystore.resource = reportportal-client-v2.jks
```

Comment:
Actually, keystore-related parameter are not required now. Could you please highlight it in doc